conformance/lang/expressions/literals/floating_point_literal.balt

```diff
@@ -0,0 +1,6159 @@
+Test-Case: output
+Description: Test decimal floating point number with float type descriptor.
+Labels: float, FloatingPointTypeSuffix
+
+function init() {
```

Comment:
There are lengthy functions. Can we group similar cases together & break this down to smaller tests

Reply:
eg: largest and smallest representable floating-point values, or testing values with many digits after the decimal point, etc. Break into separate functions when possible

Reply:
[7bdcae0](https://github.com/ballerina-platform/ballerina-spec/pull/947/commits/7bdcae05efa9f1bb65d49a7b43fd747ff122f625)